src/index.ts

```diff
@@ -1,7 +1,10 @@
+import { AttemptOptions, retry, sleep } from '@lifeomic/attempt';
+const DEFAULT_RETRY_DELAY_MS = 50;
```

Comment:
I don't think we need a default delay. Let's keep it as an option if somebody wants to set it but I'd use the library's default otherwise.